### PR TITLE
Added  initialDelayCooldownPeriod parameter to KEDA ScaledObject Doc

### PR DESCRIPTION
**Pull request by xrwang8**
This commit introduces the  parameter for KEDA's ScaledObject resource. This parameter sets a delay before the  activates after the initial creation of the ScaledObject. By default, this delay is 0 seconds. If configured, e.g., to 120 seconds, the cooldown period only begins after the ScaledObject has been active for the specified duration. This feature allows for more controlled scaling behavior during the initial deployment phase.

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes https://github.com/kedacore/keda/issues/5008
Relates to https://github.com/kedacore/keda/pull/5478

### CODE REVIEW

File: content/docs/2.14/concepts/scaling-deployments.md [hunk 1]
```diff
@@ -249,10 +283,11 @@ For examples of this feature see section [Scaling Modifiers](#scaling-modifiers-
 > 💡 **NOTE:** You can find all supported triggers [here](/scalers).
 
 Trigger fields:
+
 - **type**: The type of trigger to use. (Mandatory)
 - **metadata**: The configuration parameters that the trigger requires. (Mandatory)
 - **name**: Name for this trigger. This value can be used to easily distinguish this specific trigger and its metrics when consuming [Prometheus metrics](../operate/prometheus.md). By default, the name is generated from the trigger type. (Optional)
-- **useCachedMetrics**: Enables caching of metric values during polling interval (as specified in `.spec.pollingInterval`). For more information, see ["Caching Metrics"](#caching-metrics). (Values: `false`, `true`, Default: `false`, Optional)
+- **useCachedMetrics**: Enables caching of metric values during polling interval (as specified in `.spec.pollingInterval`). For more information, see [&#34;Caching Metrics&#34;](#caching-metrics). (Values: `false`, `true`, Default: `false`, Optional)
```
Comment (JorTurFer) [inline content/docs/2.14/concepts/scaling-deployments.md:290]:
```suggestion
- **useCachedMetrics**: Enables caching of metric values during polling interval (as specified in `.spec.pollingInterval`). For more information, see ["Caching Metrics"](#caching-metrics). (Values: `false`, `true`, Default: `false`, Optional)
```

**Review by JorTurFer — COMMENTED** [review]
Looking good! 😄 

Could you undo unrelated changes? 🙏 (spaces, new lines, etc)

**Review by JorTurFer — APPROVED** [review]
LGTM!

**Review by zroubalik — COMMENTED** [review]
Looking good, just please rename to `initialCooldownPeriod`

**Review by zroubalik — APPROVED** [review]

**Pull request merged**